SECP-service/src/main/resources/assets/app/views/register/register.html

```diff
@@ -18,16 +18,16 @@ <h1 id="formHeader">Sign up</h1>
                    name="email" unique-field-validation="{{verifyEmailUrl}}" ng-model="user.email" required="required" value="" />
             <span class="highlight"></span>
             <span class="bar"></span>
-            <span class="error" ng-show="frm.email.$touched && frm.email.$error.pattern">Enter a valid email.</span>
-            <span class="error" ng-show="frm.email.$error.invalidInputValue && frm.email.$dirty">Email already exists</span>
+            <span class="error" ng-show="frm.email.$touched && frm.email.$error.pattern">Enter a valid email.<br></span>
```

Comment:
if I am correct. i think the break syntax in html is 'br /' not 'br'. Please kindly verify if it looks good when you break the errors. Thanks :)

Reply:
<br> is actually preffered in HTML5 :)